### PR TITLE
Docs: Refactor Media Page and add more videos

### PR DESCRIPTION
**Pull request by nnelgxorz**
# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [x] Docs / tests

# Description

Refactors the media page to remove templating duplication, and keep data out of the JSX.

Adds more videos that I had previously seen on my Youtube feed.

# Checklist:

- [x ] My code follows the [developer guidelines of this project](../CONTRIBUTING.md)
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

File: packages/docs/src/routes/media/index.tsx [hunk 1]
```diff
@@ -2,176 +2,70 @@
...
+}
 
-      <section>
-        <h2 id="videos">
-          <a href="#videos">Videos</a>
-        </h2>
+export const ThumbnailLink = component$((props: { entry: MediaEntry }) => {
+  return (
+    <li>
+      <a href={props.entry.href} target="_blank">
```
[the remark this answers is not in the document]
Comment (manucorporat) [reply packages/docs/src/routes/media/index.tsx:14]: interest alert! can we fix?

[the remark this answers is not in the document]
Comment (nnelgxorz) [reply packages/docs/src/routes/media/index.tsx:14]: Definitely. Just adding noopener/noreferrer will fix this?

[the remark this answers is not in the document]
Comment (nnelgxorz) [reply packages/docs/src/routes/media/index.tsx]: Fixed!

File: packages/docs/src/routes/media/index.tsx [hunk 2]
```diff
@@ -2,176 +2,70 @@
...
-              <img src="http://i3.ytimg.com/vi/z14c3u9q8rI/hqdefault.jpg" aria-hidden="true" />
-              <p>Qwik JS and the future of frameworks</p>
-            </a>
-          </li>
-        </ul>
-      </section>
+export const BulletLink = component$((props: { entry: MediaEntry }) => {
+  return (
+    <li>
+      <a href={props.entry.href} target="_blank">
```
[the remark this answers is not in the document]
Comment (nnelgxorz) [reply packages/docs/src/routes/media/index.tsx]: Ficed!

**Pull request merged**